### ISSUE TITLE
Argument wrapping fails in the precense of comments

### ISSUE THREAD

**Issue by 333fred** [dotnet/roslyn#40368]
Consider this code:

```cs
public void M(object param1, object param2)
{
	M(
		1,
		// Comment
		2);
}
```

We do not show any help for adjusting how the comments are wrapped in this case. While I understand not being able to unwrap comments here (where are we supposed to put the comment, after all), changing the indentation level of the arguments should be possible. /cc @CyrusNajmabadi for thoughts on how this could be improved.

**Comment by CyrusNajmabadi** [comment]
Yes, it can def be improved.  I didn't do it because of code complexity.  Feel free to introduce a PR if you want!

**Comment by CyrusNajmabadi** [comment]
Would take a targetting community PR here.
